huxley/logging/handlers.py

```diff
@@ -15,10 +16,16 @@ def emit(self, record):
 
         try:
             self.format(record)
+            data = json.loads(record.message)
+            message = data["message"]
```

Comment:
Same thing with inlining the local variables into the `LogEntry` call. Also, nit: use single quotes.